docs/content/getting-started/basics.md

```diff
@@ -5,64 +5,66 @@ cases that it addresses:
 
 [[toc]]
 
-## Use case 1 - Dealing with backup (and restore) in a delivery workflow
+## Use case 1 - Dealing with backup (and restoration) in a delivery workflow
 
 During your project life cycle, you will need to deploy new versions on one or
 many production environments. We all know that we can't prevent problems to
-happen and ideally, you always backup your database before you upgrade.
+happen, that's why, ideally, you always backup your database before upgrading.
 
-The *DbToolsBundle* will use `pg_dump`, `mysqldump` or other with the correct parameters,
-just run:
+Get this task done quickly with:
 
 ```sh
 console db-tools:backup
 ```
 
-With this command, you will backup your current database, but the
-*DbToolsBundle* also provides a simple backups manager. At the end, it will
-give you the opportunity to clean old backups from your disk.
+*DbToolsBundle* will call the right backup program (`pg_dump`, `mysqldump` or
+other) with the correct parameters. At the end of the process, it will give you
+the opportunity to clean old backups from your disk.
 
-Now, let's say that your migrations were not so robust, and something bad happenned
-during your deployment, something really bad... You can't fix it quickly and you
-decide that a rollback is better for now.
+Now, let's say that your migrations were not so robust and something bad
+happened during your deployment, something really bad... You can't fix it
+quickly, so you decide to rollback for now.
 
-Run:
+Simply run:
 
 ```sh
 console db-tools:restore
 ```
 
-It will list you all the backups present on your disk, choose the one you want to restore!
+This command will list you all backups available on your disk. Choose the one
+you want to restore!
 
 These commands are only kinds of shortcut for `pg_dump`/`pg_restore` or
 `mysqldump`/`mysql` but much more handy to use. They find the correct binaries
 and execute them with the correct options for you.
 
-## Use case 2 - Anonymizing while getting back production data on your local environment
+## Use case 2 - Working with anonymized production data on your local environment
```

Comment:
Not sure about this one,  what I wanted to express was "How to anonymize sensitive data coming from your prod"
Here, with your suggestion, I understand that data are already anonymized.

Reply:
yes i agree, the current version is not so good.

Maybe something like : "Anonymizing production data when using them on your local environment"